packages/ERTP/src/types.js

```diff
@@ -97,6 +97,19 @@
  * to set subtraction.
  */
 
+/**
+ * @typedef {Object} DisplayInfo
+ * @property {number} decimalPlaces
+ *   Tells the display software how many decimal places to move the
+ *   decimal over to the left, or in other words, which position corresponds to whole
+ *   numbers. We require fungible digital assets to be represented in
+ *   integers, in the smallest unit (i.e. USD might be represented in mill,
+ *   a thousandth of a dollar. In that case, `decimalPlaces` would be 3.)
+ *   For non-fungible digital assets, this should be left as undefined.
```

Comment:
`DisplayInfo` should be expected to grow over time as one would expect of an options bag. Each use would only contain properties appropriate for the kind of amount math it is used on. Rather than having `decimalPlaces` be `undefined`, the property should be optional. AFAICT your `assertDisplayInfo` already allows it to be absent so it is just an issue of how you type and describe it here.

Reply:
ah, yep, that is what I intended. Will change